Assets/Scripts/Character/CharacterController.cs

```diff
@@ -0,0 +1,121 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using MyBox;
+
+public class CharacterController : MonoBehaviour {
+    [SerializeField] [MustBeAssigned] private Rigidbody _rigidbody;
+    [SerializeField] [MustBeAssigned] private Transform _bodyTransform;
+    [SerializeField] [MustBeAssigned] private Transform _cameraTransform;
+    [SerializeField] [MustBeAssigned] private Transform _groundCheckTransform;
+    [SerializeField] [MustBeAssigned] private LayerMask _groundLayer;
+    [SerializeField] [MustBeAssigned] private Animation _watchArmAnimation;
+
+
+    [SerializeField] [ReadOnly] private float pitchDegree = 0f;
+    [SerializeField] [ReadOnly] private float yawDegree = 0f;
+    [SerializeField] [ReadOnly] private Vector3 velocity = Vector3.zero; // For debugging purposes
+    [SerializeField] [ReadOnly] private Vector3 moveDirection = Vector3.zero;
+
+    [PositiveValueOnly] public float SENSITIVITY = 1f;  // Mouse sensitivity
```

Comment:
Will rename to lowercase.

I left it public because sensitivity could be changed from settings, and sprintMultiplier may be changed from the input script when sprint key is down.